packages/docusaurus-theme-classic/src/theme/SiteMetadata/index.tsx

```diff
@@ -33,16 +34,23 @@ function AlternateLangHeaders(): JSX.Element {
   return (
     <Head>
       {Object.entries(localeConfigs).map(([locale, {htmlLang}]) => (
-        <link
-          key={locale}
-          rel="alternate"
-          href={alternatePageUtils.createUrl({
-            locale,
-            fullyQualified: true,
-          })}
-          hrefLang={htmlLang}
-        />
+       
+          <link
+            key={locale}
+            rel="alternate"
+            href={alternatePageUtils.createUrl({
+              locale,
+              fullyQualified: true,
+            })}
+            hrefLang={htmlLang}
+          />
       ))}
+      {Object.entries(localeConfigs).map(([locale]) => (
+            <meta property="og:locale:alternate" content={locale} />
+      ))}
+
+      <meta property="og:locale" content={i18n.defaultLocale} />
```

Comment:
```suggestion
      <meta property="og:locale" content={defaultLocale} />
```

Variable defaultLocale is already in the function scope